warplib/src/conversions/integer_conversions.cairo

```diff
@@ -36,3 +39,34 @@ fn bool_into_felt252(val: bool) -> felt252 {
         0
     }
 }
+
+// Cairo-1.0 corelib code from the RC
```

Comment:
RC is now merged, could it be deleted then?

Reply:
Oh, not that RC :( 1.1 RC

Reply:
Oh wait, maybe that one! Let me check once again

Reply:
Please add `TODO: remove [...]` just to be explicit

Reply:
Here you go